.changeset/chilly-zebras-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-anywidget": major
```

Comment:
We automate releases (and versioning) with [changesets](https://github.com/changesets/changesets). When we create a release, changesets collects all the .md files from `.changeset` and determines which packages need to be released and with what versions.

For this kind of CLI, I don't think versioning is super important (what is a "breaking" change) but I'd probably make changing the location of `js` a minor changeset (since we are pre-1.0).

```suggestion
"create-anywidget": minor
```